packages/backend/migrations/20230927223235_add_additionalFields_quotes.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * @param { import("knex").Knex } knex
+ * @returns { Promise<void> }
+ */
+exports.up = function (knex) {
+  return knex.schema.alterTable('quotes', (table) => {
+    table.json('additionalFields').nullable()
```

Comment:
Is this meant to be temporary until ilp related stuff is migrated in https://github.com/interledger/rafiki/issues/2000?

otherwise I'm concerned about the schemaless-ness of jsonb

Reply:
Long term, temporary. I need to determine what non generic items are needed for the mojaloop quote, and decide how to store those as well. I imagine we end up having a separate table for method-specific quoting/transfer items